recipes/faiss/config.yml

```diff
@@ -0,0 +1,5 @@
+versions:
+  "1.8.0":
+    folder: all
+  "1.7.4":
+    folder: all
```

Comment:
I would only keep the latest version unless you specifically need the older one for some reason.